e2e/jobsubmissions/input/xyz.hcl

```diff
@@ -0,0 +1,36 @@
+variable "X" {
+  type = string
+}
+
+variable "Y" {
+  type = number
+}
+
+variable "Z" {
+  type = bool
+}
+
```

Comment:
Not sure if it's relevant as a test case here, but there's also a shorter version for declaring variables:

```hcl
variables {
  A = "B"
}
```

Reply:
Yeah these kinds of variables that have the value set directly in the original job are "invisible", at least not without doing some work with the hcl parser.